base/util.jl

```diff
@@ -506,9 +506,12 @@ order to function correctly with the keyword outer constructor.
     `Base.@kwdef` for parametric structs, and structs with supertypes
     requires at least Julia 1.1.
 
+!!! compat "Julia 1.9"
+    This macro was added to the public API and exported in Julia 1.9
```

Comment:
```suggestion
    This macro was added to the public API and exported in Julia 1.9.
```

Reply:
Maybe:
```suggestion
    This macro is exported as of Julia 1.9.
```
? Since it's not like a patch release to a minor version earlier will break it at this point. This language also matches https://github.com/JuliaLang/julia/blob/bcda258e070dcae432b995f4382e1f2a9c4b505a/base/missing.jl#L37 and https://github.com/JuliaLang/julia/blob/bcda258e070dcae432b995f4382e1f2a9c4b505a/base/reflection.jl#L1880